src/java/org/apache/cassandra/service/accord/AccordExecutorAbstractLockLoop.java

```diff
@@ -278,7 +279,15 @@ protected void runWithoutLock()
     public void shutdown()
     {
         shutdown = true;
-        notifyWorkExclusive();
+        notifyWork();
+        lock.lock();
+        try
```

Comment:
think this `try/finally` isn't needed?

Reply:
Oh yes, leftover it looks like, not sure how that happened